### PR TITLE
Update wsdl to comply with the latest ws-security policy

### PR DESCRIPTION
**Pull request by mhpnguyen**
Reviewers: Can you please review?

### CODE REVIEW

**Comment by jsmith2** [comment]
Reviewed, no comments, endorsing for merge.

File: CORERule2.2.0/src/main/resources/wsdl/AdapterCORERule2.2.0RealTimeSecured.wsdl [hunk 1]
```diff
@@ -94,6 +95,7 @@
                                 <sp:Basic192Sha256Rsa15 />
                                 <sp:Basic128Sha256Rsa15 />
                                 <sp:TripleDesSha256Rsa15 />
+                                </wsp:ExactlyOne>
```
Comment (sjarral112) [inline CORERule2.2.0/src/main/resources/wsdl/AdapterCORERule2.2.0RealTimeSecured.wsdl]: Need Indentations here..

File: CORERule2.2.0/src/main/resources/wsdl/CORERule2.2.0GenericBatch.wsdl [hunk 2]
```diff
@@ -95,6 +96,7 @@
                                 <sp:Basic192Sha256Rsa15 />
                                 <sp:Basic128Sha256Rsa15 />
                                 <sp:TripleDesSha256Rsa15 />
+                                </wsp:ExactlyOne>
```
Comment (sjarral112) [inline CORERule2.2.0/src/main/resources/wsdl/CORERule2.2.0GenericBatch.wsdl]: Need indentations for policy and exactlyone

File: DocumentQuery/src/main/resources/wsdl/NhinDocQuery.wsdl [hunk 3]
```diff
@@ -238,6 +239,7 @@
                 <sp:Basic192Sha256Rsa15/>
                 <sp:Basic128Sha256Rsa15/>
                 <sp:TripleDesSha256Rsa15/>
+                </wsp:ExactlyOne>
```
Comment (sjarral112) [inline DocumentQuery/src/main/resources/wsdl/NhinDocQuery.wsdl]: need indentations here

File: DocumentRetrieve/src/main/resources/wsdl/NhinDocRetrieve.wsdl [hunk 4]
```diff
@@ -182,6 +183,7 @@
                                 <sp:Basic192Sha256Rsa15/>
                                 <sp:Basic128Sha256Rsa15/>
                                 <sp:TripleDesSha256Rsa15/>
+                                </wsp:ExactlyOne>
```
Comment (sjarral112) [inline DocumentRetrieve/src/main/resources/wsdl/NhinDocRetrieve.wsdl]: need indentations here

File: DocumentRetrieve/src/main/resources/wsdl/NhinDocRetrieve30.wsdl [hunk 5]
```diff
@@ -179,6 +180,7 @@
                                 <sp:Basic192Sha256Rsa15/>
                                 <sp:Basic128Sha256Rsa15/>
                                 <sp:TripleDesSha256Rsa15/>
+                                </wsp:ExactlyOne>
```
Comment (sjarral112) [inline DocumentRetrieve/src/main/resources/wsdl/NhinDocRetrieve30.wsdl]: Need indentation here

File: DocumentSubmission/src/main/resources/wsdl/NhinXDRDeferredRequest.wsdl [hunk 6]
```diff
@@ -80,6 +81,7 @@
                                 <sp:Basic192Sha256Rsa15 />
                                 <sp:Basic128Sha256Rsa15 />
                                 <sp:TripleDesSha256Rsa15 />
+                                </wsp:ExactlyOne>
```
Comment (sjarral112) [inline DocumentSubmission/src/main/resources/wsdl/NhinXDRDeferredRequest.wsdl]: Need indentation here

File: DocumentSubmission/src/main/resources/wsdl/NhinXDRDeferredRequest20.wsdl [hunk 7]
```diff
@@ -96,6 +97,7 @@
                                 <sp:Basic192Sha256Rsa15 />
                                 <sp:Basic128Sha256Rsa15 />
                                 <sp:TripleDesSha256Rsa15 />
+                                </wsp:ExactlyOne>
```
Comment (sjarral112) [inline DocumentSubmission/src/main/resources/wsdl/NhinXDRDeferredRequest20.wsdl]: Need indentations here

File: DocumentSubmission/src/main/resources/wsdl/NhinXDRDeferredResponse.wsdl [hunk 8]
```diff
@@ -80,6 +81,7 @@
                                 <sp:Basic192Sha256Rsa15 />
                                 <sp:Basic128Sha256Rsa15 />
                                 <sp:TripleDesSha256Rsa15 />
+                                </wsp:ExactlyOne>
```
Comment (sjarral112) [inline DocumentSubmission/src/main/resources/wsdl/NhinXDRDeferredResponse.wsdl]: Need indentations here

File: DocumentSubmission/src/main/resources/wsdl/NhinXDRDeferredResponse20.wsdl [hunk 9]
```diff
@@ -80,6 +81,7 @@
                                 <sp:Basic192Sha256Rsa15 />
                                 <sp:Basic128Sha256Rsa15 />
                                 <sp:TripleDesSha256Rsa15 />
+                                </wsp:ExactlyOne>
```
Comment (sjarral112) [inline DocumentSubmission/src/main/resources/wsdl/NhinXDRDeferredResponse20.wsdl]: Need indentations here

File: PatientDiscovery/src/main/resources/wsdl/NhinPatientDiscoveryDeferredRequest.wsdl [hunk 10]
```diff
@@ -80,6 +81,7 @@
                                 <sp:Basic192Sha256Rsa15 />
                                 <sp:Basic128Sha256Rsa15 />
                                 <sp:TripleDesSha256Rsa15 />
+                                </wsp:ExactlyOne>
```
Comment (sjarral112) [inline PatientDiscovery/src/main/resources/wsdl/NhinPatientDiscoveryDeferredRequest.wsdl]: Need indentations here

File: PatientDiscovery/src/main/resources/wsdl/NhinPatientDiscoveryDeferredResponse.wsdl [hunk 11]
```diff
@@ -81,6 +82,7 @@
                                 <sp:Basic192Sha256Rsa15 />
                                 <sp:Basic128Sha256Rsa15 />
                                 <sp:TripleDesSha256Rsa15 />
+                                </wsp:ExactlyOne>
```
Comment (sjarral112) [inline PatientDiscovery/src/main/resources/wsdl/NhinPatientDiscoveryDeferredResponse.wsdl]: Need indentations here

**Comment by sjarral112** [comment]
Endorsed to merge !

**Pull request merged**